ui/raidboss/raidboss_options.ts

```diff
@@ -50,7 +52,11 @@ type RaidbossNonConfigOptions = {
   PerTriggerAutoConfig: PerTriggerAutoConfig;
   PerTriggerOptions: PerTriggerOptions;
   PerZoneTimelineConfig: PerZoneTimelineConfig;
+
+  // loaded builtin triggers and user triggers
   Triggers: LooseTriggerSet[];
+  LoadedTriggers: LoadedTriggerSet[];
```

Comment:
because we set `filename` in `raidboss_config.ts` https://github.com/quisquous/cactbot/blob/4fdec62f69fc076f030e68b6017045d4e46c8bb1/ui/raidboss/raidboss_config.ts#L1456

Reply:
What do you think about adding `filename?` to the trigger set type instead? Arguably it's already there, as you can do stuff like <https://github.com/quisquous/cactbot-user/blob/main/raidboss.js#L309>.

Reply:
sounds a good idea, maybe seprated PR? This refactoring PR may be better to keep behavior not changed.

Reply:
I think it's less behavior change to do add that field there, since you don't have to expose extra things in `Options` to pass typed objects around.  Then you can remove that type entirely and just use `LooseTriggerSet`.  The field has arguably been there already, since the code adds it if it's not there.

Reply:
Ok